app/models/agents/webhook_agent.rb

```diff
@@ -88,7 +89,11 @@ def receive_web_request(params, method, format)
         create_event(payload: payload)
       end
 
-      [interpolated(params)['response'] || 'Event Created', code]
+      if interpolated['response_headers'].presence
```

Comment:
I see, it kind of feels like the return value should be a hash, but that does does not have to happen in this PR.